front/src/reducers/osrdconf/types.ts

```diff
@@ -159,3 +161,5 @@ export type PacedTrainResponseWithPacedTrainId = Omit<PacedTrainResponse, 'id'>
 export type TimetableItemWithTimetableId =
   | TrainScheduleResultWithTrainId
   | PacedTrainResponseWithPacedTrainId;
+
+export type TimetableItemBase = TrainScheduleBase | PacedTrain;
```

Comment:
`TrainScheduleBase` will soon be renamed `TrainSchedule` so we can remove the `Base` I think
```suggestion
export type TimetableItem = TrainScheduleBase | PacedTrain;
```

Reply:
Done